backend/geonature/core/auth/routes.py

```diff
@@ -112,6 +112,15 @@
                 "id_organisme": organism_id,
             }
             response.set_cookie("current_user", str(current_user), expires=cookie_exp)
+
+            # Log the user in
+            user = db.session.execute(
+                sa.select(models.User)
+                .where(models.User.identifiant == current_user["user_login"])
+                .where(models.User.filter_by_app())
+            ).scalar_one()
+            login_user(user)
```

Comment:
**Suggestion:** Consider handling the case where the user is not found in the database. This will prevent the `login_user` function from being called with a `None` value. [possible issue]
```suggestion
            user = models.User.query.filter_by(identifiant=current_user["user_login"]).filter_by_app().one_or_none()
            if user is not None:
                login_user(user)
            else:
                # handle the case where the user is not found
```